<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recalibrate</class>
 <widget class="QDialog" name="recalibrate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="calibrate_img">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择标定图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calibrate_file">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择输出文件夹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calibrate_begin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>224</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入文件名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
